File: python/3rd_week/latihanpraktikum3_py/Disk.py
class Disk:
    # implementasi class Disk

    # atribut
    diskType = ""
    diskCapacity = ""
    diskPrice = 0

    # contructor
    def __init__(self):
        pass

    # get and set disk type
    def setDiskType(self, diskType):
        self.diskType = diskType

    def getDiskType(self):
        return self.diskType    
    
    # get and set disk capcity
    def setDiskCapacity(self, diskCapacity):
        self.diskCapacity = diskCapacity
    
    def getDiskCapacity(self):
        return self.diskCapacity

    # get and set disk price
    def setDiskPrice(self, diskPrice):
        self.diskPrice = diskPrice
    
    def getDiskPrice(self):
        return self.diskPrice

    # method print data disk
    def printDisk(self):
        print("-Data Disk")
        print(f" Type           : {self.diskType}")
        print(f" Kapasitas Disk : {self.diskCapacity}")
        print(f" Harga Disk     : {self.diskPrice}")
        print("--------------------------------------")
    

File: java/javadpbo/src/com/sem4dpbo/week3/praktikum3/latihanpraktikum3_java/Disk.java
package com.sem4dpbo.week3.praktikum3.latihanpraktikum3_java;

public class Disk {
    private String type, diskCapacity;
    private Long diskPrice;

//    constructor
    Disk(){}
    Disk(String type, String diskCapacity, Long diskPrice){
        this.type = type;
        this.diskCapacity = diskCapacity;
        this.diskPrice = diskPrice;
    }

    //    set and get type
    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    //set and get disk capacity
    public String getDiskCapacity() {
        return diskCapacity;
    }

    public void setDiskCapacity(String diskCapacity) {
        this.diskCapacity = diskCapacity;
    }

    //set and get disk price
    public Long getDiskPrice() {
        return diskPrice;
    }

    public void setDiskPrice(Long diskPrice) {
        this.diskPrice = diskPrice;
    }

    public void printDisk(){
        System.out.println("-Data Disk");
        System.out.println(" Type          : "+this.type);
        System.out.println(" Disk Capacity : "+this.diskCapacity);
        System.out.println(" Disk Price    : "+this.diskPrice);
        System.out.println("------------------------------------");
    }

}
